app/controllers/signup_controller.rb

```diff
@@ -12,6 +12,9 @@ def new
   end
 
   def create
+    puts session[:app_user]
```

Comment:
Is this code just for testing?

Reply:
Yeah that was just me trying to figure out why session[:app_user] was working and session['app_user'] wasn't. I will be sure to remove than line.